TCS34725.h

```diff
@@ -150,7 +150,19 @@ class TCS34725_
         return b;
     }
 
-    const Color& color() const { return clr; }
+    const Color& color() const {
+      if (raw_data.c == 0) clr.r = clr.g = clr.b = 0;
```

Comment:
You can't modify the member value in the `const` method

Reply:
This makes compile error

Reply:
> You should check the compilation before opening the pull request.

I did, but since my program does not use this function the compiler did not complain. I should have used the example. :-(
Probably, it's easier for you to fix it than me doing that? I appreciate learning a lot, but it's probably annoying for you...